tools/travis_script.sh

```diff
@@ -67,6 +67,8 @@ fi
 
 retry pip install -q tifffile
 
+pip install 'dask[array]>=0.5.0'
```

Comment:
Try moving this to this [line](https://github.com/scikit-image/scikit-image/blob/master/tools/travis_before_install.sh#L56)